test/LocalNeighborhood/test_ln.py

```diff
@@ -1,46 +1,68 @@
+from src.util import compare_files
+from src.local_neighborhood import LocalNeighborhood
 import pytest
-import sys
+import shutil
```

Comment:
To echo what we discussed yesterday with Erik's code review, you wouldn't need to import `shutil` or `compare_files`. No need to make this change, but want to flag it.